README.md

```diff
@@ -94,6 +87,11 @@ As a first step you'll need to:
    - In the **Delegated permissions** section, ensure that the right permissions are checked: **openid**, **email**, **profile**, **offline_access**, **User.Read**, **Group.Read.All**, **User.ReadBasic.All**. Use the search box if necessary.
    - Select the **Add permissions** button
 
+1. Once you've run the script, be sure to follow the manual steps. Indeed Azure AD PowerShell does not create an app which audience is Work or School + personal accounts, even if this registration is already possible from the Azure portal:
```

Comment:
And that is no longer necessary. Anyway this won't work with MSAs.

Reply:
its AnyOrg now

---
In reply to: [271644229](https://github.com/Azure-Samples/active-directory-dotnet-admin-restricted-scopes-v2/pull/16#discussion_r271644229) [](ancestors = 271644229)

Reply:
then the manual step is no longer necessary!